### PR TITLE
[bitnami/ghost] Fix host when using ingress with external TLS termination

### PR DESCRIPTION
**Pull request by mwoelk**
**Description of the change**

Gives `ghostHost` precedence over `ingress.hostname` when defining the `ghost.host` variable helper.

**Benefits**

This way, the host can be set independently / override the hostname from ingress. 
This is necessary when using external TLS termination. In this case, we need to provide a `https://`-prefix for the host which is not possible through an ingress hostname.

**Possible drawbacks**

none

**Applicable issues**

none

**Additional information**

none

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by mwoelk** [comment]
Bumped the minor version instead of patch version because the precedence change might require a check to see if `ghostHost` is set correctly if both `ghostHost` and `ingress.hostname` are provided.

**Comment by mwoelk** [comment]
Problem is that this messes with the live/readiness check. As they are performed via http but Ghost redirects to https.
Maybe it's a solve to change live/readiness to a tcp check on the http port?

File: bitnami/ghost/Chart.yaml [hunk 1]
```diff
@@ -33,4 +33,4 @@ name: ghost
 sources:
   - https://github.com/bitnami/bitnami-docker-ghost
   - http://www.ghost.org/
-version: 15.1.2
+version: 15.2.1
```
Comment (miguelaeh) [inline bitnami/ghost/Chart.yaml:36]:
```suggestion
version: 15.1.3
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you for the PR!

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Hi, 
It seems you need to bump the version again after rebasing master.

**Pull request closed without merging**